From: Nicolas Dandrimont <olasd@debian.org>
Date: Wed, 20 Nov 2019 15:16:37 +0100
Subject: Drop coverage options from pytest

---
 pytest.ini | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/pytest.ini b/pytest.ini
index ab68748..d3b0326 100644
--- a/pytest.ini
+++ b/pytest.ini
@@ -18,14 +18,6 @@ addopts =
   # https://docs.pytest.org/en/stable/doctest.html
   --doctest-modules
 
-  # Pre-load the `pytest-cov` plugin early:
-  -p pytest_cov
-
-  # `pytest-cov`:
-  --cov
-  --cov-config=.coveragerc
-  --cov-context=test
-
   # Fail on config parsing warnings:
   # --strict-config
 
